lib/dfe/reference_data/itt/cycles.rb

```diff
@@ -113,6 +113,18 @@ def self.make_local_time(*args)
               christmas: Date.new(2024, 12, 16)..Date.new(2025, 1, 3),
               easter: Date.new(2025, 4, 7)..Date.new(2025, 4, 21)
             }
+          },
+          '2025-2026' => {
+            find_opens: make_local_time(2025, 10, 7, 9), # First Tuesday of October
+            apply_opens: make_local_time(2024, 10, 14, 9), # Second Tuesday of October
+            apply_1_deadline: nil, # This deadline is not applicable for this cycle
+            apply_2_deadline: make_local_time(2026, 9, 22, 18), # 3rd Tuesday in September
```

Comment:
"3rd Tuesday in September. This is the final and only deadline to send ITT applications in the cycle."